EXT_BeamPhysics.md

```diff
@@ -161,12 +161,12 @@ The following records store data on a particle-by-particle basis.
           - `-c.t`: `z` component is the phase space coordinate conjugate to an energy based "pz".
           - `z`: `z` component is the true longitudinal coordinate.
 
-- `refTime/`
+- `refTime`
     - Type: Optional *(float)*
     - Description: The reference particle time.
     Note that the reference time is a function of **s** and therefore can be different for different particles.
 
-- `s-Position/`
+- `s-Position`
```

Comment:
Just a style question: really `-` and camelCase in this name?

Reply:
I was not thinking. To be consistent that should be `sPosition`.  I will make the change. The `time-refTime` is to emphasize that this is a time difference. Do you have an alternative name?

Reply:
Maybe `refTimeDifference`?

Reply:
Now `sPosition`